pandas/io/sql.py

```diff
@@ -371,6 +379,9 @@ def read_sql_query(
     chunksize : int, default None
         If specified, return an iterator where `chunksize` is the number of
         rows to include in each chunk.
+    dtype : Type name or dict of columns
+        Data type for data or columns. E.g. np.float64 or
+        {‘a’: np.float64, ‘b’: np.int32, ‘c’: ‘Int64’}
```

Comment:
need a versionadded 1.3 here. ok to add in next PR

Reply:
Sorry, I see i didn't commit that change. But will indeed add it to the follow on